docs/DeveloperGuide.md

```diff
@@ -1,34 +1,152 @@
 # Developer Guide
```

Comment:
Would it be good to include a simple description of what the application is supposed to do so that the developer gets a clear idea of the purpose?